sdk/cosmos/azure-cosmos/samples/autoscale_throughput_management.py

```diff
@@ -0,0 +1,274 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: autoscale_throughput_management.py
+
+DESCRIPTION:
+    This sample demonstrates how to manage autoscale throughput settings for
+    Azure Cosmos DB databases and containers. Autoscale allows you to automatically
+    scale throughput based on usage, providing cost optimization and performance flexibility.
+
+    Key concepts covered:
+    - Creating databases and containers with autoscale throughput
+    - Reading autoscale throughput settings
+    - Updating autoscale maximum throughput
+    - Understanding autoscale increment percentage
+
+USAGE:
+    python autoscale_throughput_management.py
+
+    Set the environment variables with your own values before running:
+    1) ACCOUNT_HOST - the Cosmos DB account endpoint
+    2) ACCOUNT_KEY - the Cosmos DB account primary key
+"""
+
+import azure.cosmos.cosmos_client as cosmos_client
+import azure.cosmos.exceptions as exceptions
+from azure.cosmos.partition_key import PartitionKey
+from azure.cosmos import ThroughputProperties
+
+import config
+
+# ----------------------------------------------------------------------------------------------------------
+# Prerequisites -
+#
+# 1. An Azure Cosmos account -
+#    https://docs.microsoft.com/azure/cosmos-db/create-sql-api-python#create-a-database-account
+#
+# 2. Microsoft Azure Cosmos PyPi package -
+#    https://pypi.python.org/pypi/azure-cosmos/
+# ----------------------------------------------------------------------------------------------------------
+# Sample - demonstrates autoscale throughput management for databases and containers
+# ----------------------------------------------------------------------------------------------------------
+# Note -
+#
+# Running this sample will create (and delete) multiple Containers on your account.
+# Each time a Container is created the account will be billed for 1 hour of usage based on
+# the provisioned throughput (RU/s) of that account.
+# ----------------------------------------------------------------------------------------------------------
+
+HOST = config.settings['host']
+MASTER_KEY = config.settings['master_key']
+DATABASE_ID = config.settings['database_id']
+CONTAINER_ID = config.settings['container_id']
+
+
+def create_database_with_autoscale(client, database_id):
+    """
+    Create a database with autoscale throughput.
```

Comment:
we generally want to not recommend this - shared throughput has caused issues with customers in the past. let's add this to both samples please!
```suggestion
    Create a database with autoscale throughput. Setting throughput settings, like autoscale, on a database level is *not* recommended, and should only be done if you are aware of the implications of shared throughput across containers.
```